openshift-ci/scripts/oci-model-registry-deploy.sh

```diff
@@ -62,13 +91,15 @@ check_deployment_availability() {
 
         # Check if the deployment is available
         if [[ $deployment_status != "" ]]; then
+            echo "✔ Success: Deployment $deployment is available"
             echo "✔ Success: Deployment $deployment is available"
```

Comment:
in this file, there appears to be multiple repeated echo statement lines introduced with this PR. Is this intentional?